### ISSUE TITLE
Need to force to export files

### ISSUE THREAD

**Issue by Zuhaitz3** [odeke-em/drive#885]
Hi all,

When I do "drive pull -export odt -explicitly-export _my_file_" it says: 

_"my_file" is set to be ignored yet is being processed. Use `force` to override this._

Is this the expected behaviour? Do I need to force to export any file?

**Comment by ManuCart** [comment]
Same way for me. I don't no why

**Comment by ikwyl6** [comment]
I agree. If you specify `-export`, why do you need to `-force` it or `-explicitly-export` it? Seems redundant. 

As well, I don't know why you would need to do `-same-exports-dir` and `-exports-dir` as it seems redundant to me when you want to export to the same directory. For drive to export a docx file, I need to do `-exports-dir .`

**Comment by ikwyl6** [comment]
Hi @odeke-em can you comment on the above? Each time I read the ```-explicitly-export``` flag, I still don't know why you need to explicitly export something when you have already supplied the ```-export``` flag..
I think the ```-explicitly-export``` flag can be removed, can't it?
